src/components/Authz/Credentials.js

```diff
@@ -16,152 +14,57 @@ export function Credentials({ ...props }) {
 	const resources = useSelector(state => state?.auth?.resources);
 	const resource = 'seacat:credentials:access'; // Resource required to access the Credentials List Screen
 
-	const { t } = useTranslation();
-
-	const apiPath = props.apiPath ?? 'seacat-auth';
-
-	const credentials_ids = Array.isArray(props.credentials_ids) ? props.credentials_ids : [props.credentials_ids];
-
 	// Validation on props.app
 	if (props.app == undefined) {
-		return renderPlainCredentials(credentials_ids);
+		return renderPlainCredentials(props.credentials_ids);
 	}
 
-	const CredentialsAPI = props.app.axiosCreate(apiPath);
-
 	const cleanupTime = props.cleanupTime ?? 1000 * 60 * 60 * 24; // 24 hrs
```

Comment:
I also don't know if it's worth having as a prop `cleanupTime` because we don't add it anywhere and I could pass it on directly for `saveUsernamesToLS`